app/models/bgs_dependents/adult_child_attending_school.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module BGSDependents
+  class AdultChildAttendingSchool < Base
+    attribute :first, String
+    attribute :middle, String
+    attribute :last, String
+    attribute :suffix, String
+    attribute :ssn, String
+    attribute :birth_date, String
+    attribute :ever_married_ind, String
+
+    def initialize(dependents_application)
+      @dependents_application = dependents_application
+      @was_married = @dependents_application.dig('student_address_marriage_tuition', 'was_married')
+      @name_and_ssn = @dependents_application['student_name_and_ssn']
```

Comment:
What form does this take? Having the word 'and' in a variable is usually a good sign it's doing too much and can be broken up.

Reply:
We get back this:
```
{"full_name"=>{"first"=>"name", "middle"=>"middle name", "last"=>"my last name", "suffix"=>"II"}, "ssn"=>"removed_this", "birth_date"=>"2001-03-03"}
```

I just mirrored what it's called in the params we were sent.

Reply:
I broke up the hash. I used dig in optional fields and added validation.